.storybook/themes/news.scss

```diff
@@ -633,6 +633,46 @@
 				right-rail-advanced-main-right-rail-bottom: (
 					grid-area: main-right-rail-bottom,
 				),
+				share-bar: (
+					box-shadow: (
+						0 4px 8px 0 rgba(0, 0, 0, 0.2),
+						0 6px 20px 0 rgba(0, 0, 0, 0.19),
+					),
+					border-radius: 0 5px 5px 0,
+					display: flex,
+					left: 0,
+					padding: 9px 0,
+					position: fixed,
+					top: 200px,
+					width: 44px,
+					components: (
+						button-medium: (
```

Comment:
We want to be using a link here. it looks like we're overriding the inherent buttons styles anyway. and since this is redirecting we'd want to use an a tag anyway. think you'll have an easier time with padding, etc, eg: 


					components: (
						link: (
							padding: var(--global-spacing-2),
						),
						icon: (
							fill: var(--color-primary),
						),
					),